Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ProjectionContentGraph.php

```diff
@@ -76,6 +76,53 @@ public function findParentNode(
         return $nodeRow ? NodeRecord::fromDatabaseRow($nodeRow) : null;
     }
 
+    /**
+     * @throws DBALException
+     * @throws \Doctrine\DBAL\Driver\Exception
+     */
+    public function findParentNodeRecordByOriginInDimensionSpacePoint(
```

Comment:
can we add a short diagram what we actually query here? :-) I find it hard to understand why both OriginDSP and DSP are given - shouldn't this be unique just giving DSP? -> so a description explaining this would be nice

Reply:
It's already explained in the query, don't really know what else to add ^^